src/amo/utils/server.js

```diff
@@ -59,3 +67,23 @@ export const viewFrontendVersionHandler = ({
     });
   };
 };
+
+export const viewHeartbeatHandler = ({
+  _config = config,
+  _fetch = fetch,
+}: ViewHeartbeatHandlerParams = {}): ExpressHandler => {
+  const apiURL = `${_config.get('apiHost')}${_config.get(
+    'apiPath',
+  )}${_config.get('apiVersion')}/site/?disable_caching`;
+  return async (req: typeof $Request, res: typeof $Response) => {
+    const response = await _fetch(apiURL);
```

Comment:
My assumption is that if `_fetch` throws an exception (as opposed to returning a non-ok response), then the entire request will return a non-ok response code, which is pretty much what we want, so I'm not sure we need any error handling around this.

Reply:
Because we want to return a 500 status on failure, I added a `try/catch` around this and a test for that as well.